app/plugins/ui/commands/openwhisk-core.js

```diff
@@ -594,6 +595,12 @@ specials.actions = {
                     options.action.annotations.push({ key: 'binary', value: true })
                 }
 
+                if (options.native) {
+                    // native code blackbox action
+                    options.action.exec.kind = 'blackbox'
+                    options.action.exec.image = 'openwhisk/dockerskeleton'
```

Comment:
will it work to also create a --docker action with a text file/source?

Reply:
is that supported?

Reply:
--native foo.sh will work with this PR. what else should work? --native a.out? i tried this (i.e. a binary) with `bx wsk`, and garbage ensued.

if you could help me to understand what should and shouldn't work more precisely, that'd be helpful.